.github/pull_request_template.md

```diff
@@ -1,12 +1,11 @@
-**Fix ...**
-
 **Changes:**
 * Briefly describe changes
 
 **I have:**
 <!-- fill in with [x] -->
 - [ ] Set target branch to the correct branch, usually `dev`.
+- [ ] Rebased source branch from the target branch.
 - [ ] Ensured code is formatted, linted.
 - [ ] Cleaned up git history.
 - [ ] Ensured commit messages describe *what* was changed, and *why*.
-- [ ] Ran tests locally, checked output.
+- [ ] Ran tests locally after rebasing, checked output.
```

Comment:
Would not "- [ ] Ran tests locally after rebasing and checked output." be more appropriate?